src/doc/rustdoc/src/unstable-features.md

```diff
@@ -258,6 +258,16 @@ themselves marked as unstable. To use any of these options, pass `-Z unstable-op
 the flag in question to Rustdoc on the command-line. To do this from Cargo, you can either use the
 `RUSTDOCFLAGS` environment variable or the `cargo rustdoc` command.
 
+### `--document-hidden-items`: Show items that are `#[doc(hidden)]`
+<span id="document-hidden-items"></span>
+
+By default, `rustdoc` does not document items that are annotated with
+[`#[doc(hidden)]`](write-documentation/the-doc-attribute.html#hidden).
+
+`--document-hidden-items` causes all items to be documented as if they did not have `#[doc(hidden)]`.
```

Comment:
I'd reword it like this:

> `--document-hidden-items` includes all items with the `#[doc(hidden)]` attribute in the generated documentation.

Reply:
That's not true though? Hidden private items will still not be documented unless `--document-private-items` is specified.

Reply:
Yes? Maybe my sentence is not worded well enough but nothing in it implies that all items are documented, only items with `#[doc(hidden)]`.

Reply:
but not all items with `#[doc(hidden)]` will be documented with just `--document-hidden-items`, only those that are `pub`.

Reply:
Oh I see. Good point. What about:

> `--document-hidden-items` will include all items with the `#[doc(hidden)]` attribute in the generated documentation build, meaning that if an item is private, unless `--document-private` option is used, it will still not be displayed.

Reply:
> meaning that

this implies that the second clause logically follows from the first, but it does not.

i don't think there's a way we can say this correctly without the wording being slightly awkward.  perhaps we could include a table/diagram (sort of like a truth table) that shows what items will be documented under which combination of flags?

Reply:
A table is a very good idea!